pkg/config/log.go

```diff
@@ -20,8 +20,19 @@ import (
 	"github.com/cihub/seelog"
 )
 
+// LoggerName is a string prepended to all logs to indicate which component is logging
+type LoggerName string
```

Comment:
What's the advantage of having a type here vs. plain string constants? The documentation itself is a hint. Why not just add a string argument to `SetupLogger` calling it `prefix`? It would be much clearer what it does (imho).

Reply:
As it would be preprended to all logs I feel having a dedicated type here helps enforcing that you know what you're doing when calling the function. WDYT?

Reply:
I still feel that this is misleading. This isn't the "name of an instantiated logger", it's just a "prefix that will be prepended to each logged line".